.circleci/config.yml

```diff
@@ -14,7 +14,7 @@ jobs:
   helm-lint:
     working_directory: ~/stackstorm-ha
     docker:
-      - image: lachlanevenson/k8s-helm:v3.3.4
+      - image: lachlanevenson/k8s-helm:v3.4.2
```

Comment:
This change was triggered by: https://app.circleci.com/pipelines/github/StackStorm/stackstorm-ha/1192/workflows/9f1ccdab-bfef-4017-a98f-4b298bb49c75/jobs/4020

I'm using helm 3.5.4. e2e tests use the latest helm version, 3.6.0.